roles/neutron-common/handlers/main.yml

```diff
@@ -11,3 +11,9 @@
     - "{{ neutron.services.neutron_linuxbridge_agent }}"
     - "{{ neutron.services.neutron_metadata_agent }}"
     - "{{ neutron.services.neutron_lbaasv2_agent }}"
+
+- name: restart neutron server service
+  service: name={{ item }} state=restarted_if_running must_exist=false
+  when: restart|default('True')
```

Comment:
all occurrences uses True so that it does restart by default.  In addition, restart is overwritten to false when doing upgrading (upgrade.yml). 

please correct me if I am wrong.